orangecontrib/text/vectorization/sbert.py

```diff
@@ -0,0 +1,125 @@
+import json
+import base64
+import zlib
+import sys
+from typing import Any, List, Optional, Callable, Union
+
+import numpy as np
+
+from Orange.misc.server_embedder import ServerEmbedderCommunicator
+from Orange.util import dummy_callback
+
+# maximum document size that we still send to the server
+MAX_PACKAGE_SIZE = 3000000
+# maximum size of a chunk - when one document is longer send is as a chunk with
+# a single document
+MAX_CHUNK_SIZE = 50000
+MIN_CHUNKS = 20
+
+
+class SBERT:
+    def __init__(self) -> None:
+        self._server_communicator = _ServerCommunicator(
+            model_name='sbert',
+            max_parallel_requests=100,
+            server_url='https://apiv2.garaza.io',
+            embedder_type='text',
+        )
+
+    def __call__(
+        self, texts: List[str], callback: Callable = dummy_callback
+    ) -> List[Optional[List[float]]]:
+        """Computes embeddings for given documents.
+
+        Parameters
+        ----------
+        texts
+            A list of raw texts.
+
+        Returns
+        -------
+        An array of embeddings.
+        """
+
+        if len(texts) == 0:
+            return [None] * len(texts)
```

Comment:
I am nail picking, but it can be just `return []`
```suggestion
            return []
```